docs/sdk/quickstart-by-language/quickstart-nodejs.mdx

```diff
@@ -0,0 +1,179 @@
+can do what, based on the policy that you defined.
+
+:::tip
+If you would like to learn more about the `permit.check()` function, and how it works with different policy models, you can read
+more about it [here](/how-to/enforce-permissions/check).
+:::
+
+You can run a permission check with permit.check(), passing in 3 arguments:
+
+- `user`: a unique string id or object that identifies the user performing the action.
+- `action`: the action performed.
+- `resource`: the resource (object) the action is performed on. Can be a string or an object.
+
+:::tip
+When working with ABAC (Attribute-based Access Control), for the `user` and `resource`, you can pass an object containing
+attributes that you are looking to match against. If you want to understand permission ABAC checks better, please refer to
+[this guide](/how-to/enforce-permissions/check#attributes).
+:::
+
+In the following example we are checking that a user with the unique id `john@smith.com` can `read` a `document`.
+
+```javascript
+const permitted = await permit.check("john@permit.io", "read", "document");
+
+if (permitted) {
+  console.log("John is PERMITTED to read a document");
+} else {
+  console.log("John is NOT PERMITTED to read a document");
+}
+```
+
+## Other API configuration functionality with the SDK
+
+In this quickstart, we'll introduce you to some core Permit.io functionality using the SDK. As you'll see, the SDK is designed to
+be simple and logical, making it easy to develop and manage policies.
+
+### Creating a resource
+
+A resource is the fundamental building block of a policy. It's what the user can interact with. It could be a 'document', a 'building'
+or even a 'patient health record'.
+
+```javascript
+await permit.api.resource.create({
+  key: "repository",
+  name: "Repository",
+  actions: {
+    clone: {},
+    push: {},
+    view: {},
+  },
+});
+```
+
+### Creating a new role with permissions
+
+A role defines a set of resources and the specific actions a user can perform on those resources. A user can only perform these
+actions if they have been assigned the corresponding role.
+
+````javascript
+await permit.api.roles.create({
+    key: "manager",
+    name: "Manager",
+    permissions: [
+      "repository:clone",
+      "repository:view",
+    ],
+  });```
+````
+
+### Syncing a user to Permit
+
+The sync user function allows you to take an existing user from your authentication solution, and add them inside of Permit.
+
+```javascript
+await permit.api.syncUser({
+  key: "key",
+  email: "example@permit.io",
+  first_name: "John",
+  last_name: "Smith",
+  attributes: {},
+});
+```
+
+### Assigning a role
+
+A role assignment, as the name suggests, assigns a specific role with defined permissions to a user identity. This role assignment
+is specific to a particular tenant.
+
+```javascript
+await permit.api.roleAssignments.assign({
+  user,
+  role,
+  tenant: "default",
+});
+```
+
+## Unique SDK functions
```

Comment:
This title is redundant (especially because the section contains a single admonition